src/Whoops/Resources/css/whoops.base.css

```diff
@@ -38,7 +38,7 @@ header {
   box-sizing: border-box;
   background-color: #2a2a2a;
   padding: 35px 40px;
-  max-height: 180px;
+  max-height: 500px;
```

Comment:
Maybe this should be relative to the viewportheight with `vh` units?

Reply:
Yes, sure

Reply:
`vh` - does not give the necessary result...